pkgs/os-specific/linux/kernel/linux-zen.nix

```diff
@@ -1,28 +1,33 @@
 { lib, fetchFromGitHub, buildLinux, ... } @ args:
 
 let
-  version = "5.13.9";
-  suffix = "zen1";
+  # having the full version string here makes it easier to update
+  modDirVersion = "5.13.10-zen1";
+  parts = lib.splitString "-" modDirVersion;
+  version = lib.elemAt parts 0;
+  suffix = lib.elemAt parts 1;
+
+  numbers = lib.splitString "." version;
+  branch = "${lib.elemAt numbers 0}.${lib.elemAt numbers 1}";
```

Comment:
I think this is extremely complex for little to no gain. I would've preferred this to happen in a separate PR that isn't time critical.

Reply:
I was hoping to make it up-dateable with nix-update, which would have safed some code in the update, but this did not work.

Reply:
Auto-update for linux kernels is done with an updater script. I haven't gotten around to making it include the zen kernels though.

Complexity for update purposes should go there IMO.